sdk/core/azure-core-experimental/samples/httpx_integration/httpx_transport_sample_async.py

```diff
@@ -29,7 +29,7 @@ async def sample_abstractive_summarization_async() -> None:
     document = ["This is an example document."]
 
     async with text_analytics_client:
-        poller = await text_analytics_client.begin_abstractive_summary(document)
+        poller = await text_analytics_client.begin_abstract_summary(document)
```

Comment:
text_analytics introduced a breaking change?

Reply:
Yes they did in 5.3.0 released 6/15/2023 https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

Reply:
They are breaking changes against beta, not GA.

Maybe we should use GAed features for samples?